front/app/containers/Admin/settings/statuses/containers/index.tsx

```diff
@@ -69,7 +69,7 @@ const IdeaStatuses = ({ variant }: { variant: 'ideation' | 'proposals' }) => {
   };
 
   const isRequired = (ideaStatus: IIdeaStatusData) => {
-    return ideaStatus.attributes.locked === false;
+    return ideaStatus.attributes.locked === true;
   };
```

Comment:
A few months back this was changed from `ideaStatus.attributes.can_reorder = false` to `ideaStatus.attributes.locked = false`, which I think introduced the issue. I believe it should be `locked = true`.

Reply:
I think you are right! Thanks for catching this 🙌